webapp-django/crashstats/crashstats/templates/crashstats/topcrasher.html

```diff
@@ -33,6 +33,9 @@
                   {% endif %}
                 {% endif %}
             </ul>
+            <a href="{{ url('topcrashers:topcrashers') }}?product={{ product }}&amp;version={{ versions[0] }}&amp;date_range_type={{ date_range_type }}&amp;process_type={{ crash_type.replace('all', 'any') }}&amp;platform={{ os_name }}&amp;_facets_size={{ result_count }}&amp;days={{ days }}">
+                See the new Top Crashers report
```

Comment:
Too verbose IMHO. If it was going to be a full sentence like that you should make it `See the <a>new Top Crashers</a> report`. 
Alternatively you can just make the link a short noun. E.g. `<a>New Top Crashers report</a>`.